### DISCUSSION TITLE
Feedback Page

### DISCUSSION THREAD

**Discussion by HardikYadav99** [Anurag-Negi28/contact_list#3] [General]
hi

**Comment by Anurag-Negi28** [comment]
Hi Hardik
I am updating this as our feedback page for the project. Anyone can visit this page and add his/her feedback about the contact list project.

**Reply by HardikYadav99** [reply]
Ok Anurag
I will share the project with my friends they can give feedbacks on this.

**Reply by Anurag-Negi28** [reply]
Yeah sure

**Comment by Dhruv-Jain31** [comment]
Login page is missing. Regarding this I have raised a issue as well

**Reply by Anurag-Negi28** [reply]
Yeah the issue has been assigned

**Comment by jahesh2003** [comment]
gui is good

**Reply by Anurag-Negi28** [reply]
Thanks for your feedback

**Comment by ghost** [comment]
Nicely done

**Reply by Anurag-Negi28** [reply]
Thanks for the feedback

**Comment by SalilRichard** [comment]
gui of second version is better and simple.

**Reply by Anurag-Negi28** [reply]
Thanks for the feedback

**Comment by vrindagupta25** [comment]
hey I really liked your gui and code and everything great work guysss!!!

**Reply by Anurag-Negi28** [reply]
Thanks for the feedback

**Comment by Dhruv-Jain31** [comment]
App is nice and easy to use. Earlier I have raised a issue and you have resolved it as well. Keep up the good work 👏
